execution_workflows/IsoSCM/conf/modules.config

```diff
@@ -63,21 +63,21 @@ params {
                 If using APAeval test data, the file is provided in tests/test_data folder
                 e.g gtf_genome_file = '/home/fzhuang/apaeval/APAeval/execution_workflows/IsoSCM/../../tests/test_data/gencode_2genes_Chr_prefix.vM18.annotation.gtf'
             */
-            gtf_genome_file = '[path_to]/../../tests/test_data/gencode_2genes_Chr_prefix.vM18.annotation.gtf'
+            gtf_genome_file = '/home/fzhuang/new_apaeval/APAeval/execution_workflows/IsoSCM/../../tests/test_data/gencode_2genes_Chr_prefix.vM18.annotation.gtf'
```

Comment:
really minor comment but you should keep the `[path_to]` in this file.

Reply:
oh yes thank you! I accidentally pushed my local version as well, will revert this to the original version!